qiskit_experiments/data_processing/processor_library.py

```diff
@@ -12,39 +12,63 @@
 
 """A collection of functions that return various data processors."""
 
+from enum import Enum
+
 from qiskit.qobj.utils import MeasLevel
 
 from qiskit_experiments.data_processing.exceptions import DataProcessorError
 from qiskit_experiments.data_processing.data_processor import DataProcessor
 from qiskit_experiments.data_processing import nodes
 
 
+class ProjectorType(str, Enum):
+    """Types of projectors for data dimensionality reduction."""
+
+    SVD = "SVD"
+    ABS = "ABS"
+    REAL = "REAL"
+    IMAG = "IMAG"
+
+
 def get_processor(
     meas_level: MeasLevel = MeasLevel.CLASSIFIED,
     meas_return: str = "avg",
     normalize: bool = True,
+    dimensionality_reduction: ProjectorType = ProjectorType.SVD,
 ) -> DataProcessor:
     """Get a DataProcessor that produces a continuous signal given the options.
 
     Args:
         meas_level: The measurement level of the data to process.
         meas_return: The measurement return (single or avg) of the data to process.
         normalize: Add a data normalization node to the Kerneled data processor.
+        dimensionality_reduction: A string to represent the dimensionality reduction
```

Comment:
Could the documentation also be more specific that it means converting complex values to real? It is not clear since dimensionality can also refer to level 0 to level 1 data or single shot to average data.

Reply:
Good points. See here: 2c1c124